libs/ui/src/lib/notice/notice.component.stories.ts

```diff
@@ -0,0 +1,281 @@
+      <cvi-ng-track
+        layout="flex"
+        horizontalAlignment="justify"
+        verticalAlignment="center"
+      >
+        any content content
+      </cvi-ng-track>
+    </cvi-ng-notice>
+  `,
+});
+
+export const Default = Template.bind({});
+
+const TemplateWithError: Story<NoticeComponent> = (args: NoticeComponent) => ({
+  props: args,
+  template: `
+    <cvi-ng-notice [iconName]="iconName" [severity]="'error'">
+      <cvi-ng-track
+        layout="flex"
+        horizontalAlignment="justify"
+        verticalAlignment="center"
+      >
+        <cvi-ng-track
+          layout="flex"
+          horizontalAlignment="left"
+          verticalAlignment="top"
+          flexDirection="vertical"
+        >
+          <cvi-ng-track
+            layout="flex"
+            horizontalAlignment="justify"
+            verticalAlignment="center"
+            [gap]="4"
+          >
+            <div class="cvi-notice--dark-title">01.03.2023</div>
+            <div class="cvi-notice--dark-title underlined">
+              KMD käibedeklaratsioon
+            </div>
+          </cvi-ng-track>
+          <cvi-ng-track layout="flex" horizontalAlignment="justify" verticalAlignment="center" [gap]="4">
+            <div>23:59</div>
+            <div>Maksu- ja Tolliamet</div>
+            <cvi-ng-icon name="warning_amber" height="18"></cvi-ng-icon>
+          </cvi-ng-track>
+        </cvi-ng-track>
+        <cvi-ng-track
+          layout="flex"
+          horizontalAlignment="right"
+          verticalAlignment="center"
+          [gap]="4"
+        >
+          <cvi-ng-track
+            layout="flex"
+            horizontalAlignment="justify"
+            verticalAlignment="center"
+            [gap]="4"
+          >
+          <div>
+            138.00$
+            </div>
+            <cvi-ng-icon
+              [name]="'info'"
+              [height]="'24'"
+              [cviNgTooltip]="
+                'This tooltip is displayed when hovering, if you move mouse out of element then tooltip dissapears'
+              "
+            ></cvi-ng-icon>
+          </cvi-ng-track>
+          <cvi-ng-button> kohustust täitma </cvi-ng-button>
+        </cvi-ng-track>
+      </cvi-ng-track>
+    </cvi-ng-notice>
+  `,
+});
+
+export const WithError = TemplateWithError.bind({});
+
+const TemplateWithSuccess: Story<NoticeComponent> = (
+  args: NoticeComponent
+) => ({
+  props: args,
+  template: `
+  <cvi-ng-notice [iconName]="iconName" [severity]="'success'">
+  <cvi-ng-track
+    layout="flex"
+    horizontalAlignment="justify"
+    verticalAlignment="center"
+  >
+    <cvi-ng-track
+      layout="flex"
+      horizontalAlignment="left"
+      verticalAlignment="top"
+      flexDirection="vertical"
+    >
+      <cvi-ng-track
+        layout="flex"
+        horizontalAlignment="justify"
+        verticalAlignment="center"
+        [gap]="4"
+      >
+        <div class="cvi-notice--dark-title">01.03.2023</div>
```

Comment:
CSS classes from the library should always be encapsulated within UI components (whether library components or app's own components), not directly in views.

Reply:
Only place where there is still a class used in a view is the "cvi-notice__subtitle-icon"... not sure how to approach here, since the icon should be centered but as far as i know that is currently achievable only with some extra styling